core/os/memory.cpp

```diff
@@ -80,6 +80,7 @@ void *Memory::alloc_static(size_t p_bytes, bool p_pad_align) {
 #endif
 		return s8 + PAD_ALIGN;
 	} else {
+		// cppcheck-suppress memleak
```

Comment:
Better than silencing the warning would be fixing the issue, no? Or is it a false positive? If so it should be better explained in the code.